site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/validation/SiteValidation.java

```diff
@@ -140,7 +140,8 @@ private ValidationResponse validateCustomArgument(List<SiteAttributeConfiguratio
                                                       final Site site) {
         if (customAttributes != null) {
             for (final SiteAttributeConfiguration attr : customAttributes) {
-                if (attr.getType().equals("address") && site.getAddress() != null) {
+                if (attr.getType().equals("address") && site.getAddress() != null
```

Comment:
do we need the site.getAddress() != null check if StringUtils.hasText checks for nulls?

Reply:
Yes,  site.getAddress().checkEmptyOrNull() is checking for the contents of address object, but the site.getAddress() != null is required to ensure site.getAddress() has a non null reference.

Reply:
ahh yes silly moi..